HaxeManual/10-std.tex

```diff
@@ -1128,3 +1136,28 @@ \section{Unit Testing}
 \begin{itemize}
 	\item See the \href{https://api.haxe.org/haxe/unit/}{haxe.unit} package on the API documentation for more details.
 \end{itemize}
+
+
+\section{Haxe 3 Compatibility}
+\label{std-hx3compat}
+\since{4.0.0}
+
+Some APIs that were originally a part of the standard library have been removed in Haxe 4. To allow projects to transition from Haxe 3 more smoothly, the \href{https://github.com/HaxeFoundation/hx3compat/}{hx3compat} library still provides these APIs. To use it, simply install it and compile the project with \ic{-L hx3compat}.
+
+The full list of APIs in hx3compat is:
+
+\begin{itemize}
+	\item \type{haxe.remoting.*} Haxe remoting
+	\item \type{haxe.unit.*} Unit testing
+	\item \type{haxe.web.*} Web request classes for Haxe web servers
+	\item \type{neko.net.*} Neko web server
+	\item (JavaScript target only) jQuery and SWFObject externs
+\end{itemize}
+
+Note that some of these APIs were removed form the standard library in favour of solutions in external libraries:
+
+\begin{itemize}
+	\item \href{https://github.com/haxe-utest/utest}{haxe-utest} is a Haxe unit testing library.
```

Comment:
The haxe prefix seems slightly redundant? The lib name is just `utest`, not `haxe-utest`.

Reply:
Oh yeah, it's the GitHub org name that is `haxe-utest`.